languages/java/oso/src/main/java/com/osohq/oso/Polar.java

```diff
@@ -85,23 +85,23 @@ public void loadStr(String str) throws Exceptions.OsoException {
      * @param query String string
      * @return Query object (Enumeration of resulting variable bindings).
      */
-    public Query query(String query) throws Exceptions.OsoException {
+    public Query queryStr(String query) throws Exceptions.OsoException {
```

Comment:
This is still just called `query` in other languages.  Ok to leave for now.

Reply:
Made this fix and updated the Java query API to be the same as the others